lib/fog/core.rb

```diff
@@ -40,7 +40,6 @@
 require File.expand_path('../core/hmac', __FILE__)
 require File.expand_path('../core/logger', __FILE__)
 require File.expand_path('../core/model', __FILE__)
-require File.expand_path('../core/mock', __FILE__)
```

Comment:
No because this is an external gem, it requires a gem not a file. The gem is added to gem specs dependencies.